src/core/main.cpp

```diff
@@ -510,6 +510,15 @@ int main( int argc, char * * argv )
 		Engine::getSong()->loadProject( file_to_load );
 		printf( "done\n" );
 
+		if ( Engine::getSong()-> hasErrors() )
+		{
+			const char * summary = Engine::getSong()->errorSummary()->toStdString().c_str();
+
+			fputs( "\n", stderr );
+			fputs( summary, stderr );
+			fputs( "\n", stderr );
```

Comment:
On 01/07/2015 08:11 PM, Amadeus Folego wrote:
>
> In src/core/main.cpp
> <https://github.com/LMMS/lmms/pull/1525#discussion-diff-22603221>:
>
> > @@ -510,6 +510,15 @@ int main( int argc, char * * argv )
> >  		Engine::getSong()->loadProject( file_to_load );
> >  		printf( "done\n" );
> >  
> > +		if ( Engine::getSong()-> hasErrors() )
> > +		{
> > +			const char * summary = Engine::getSong()->errorSummary()->toStdString().c_str();
> > +
> > +			fputs( "\n", stderr );
> > +			fputs( summary, stderr );
> > +			fputs( "\n", stderr );
>
> Could someone help me here? I am clueless on what should I do, using
> |qDebug| does not make sense to me and even if you want me to use
> please tell me how to print to stderr with it.
>

Qt has great documentation. And it's a rather valuable skill for a
programmer to be able to look up things like this.

So: Rather than straight out telling you what to do, I'm going to leave
this as an exercise: google Qt documentation, look up qDebug, apply new
knowledge to problem at hand.